src/System.Text.Encoding.Extensions/tests/4.0.10.0/Encoding/ISO-8859-1.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+using System;
+using System.Text;
+
+namespace EncodingTests
+{
+    public static class ISO_8859_1
```

Comment:
I expect some of these tests (in this file and others) are going to fail on Linux and OSX.  You may need to mark the failing ones as [ActiveIssue(846, PlatformID.Linux | PlatformID.OSX)], but please investigate first to make sure any such failures are actually expected given the state of our globalization implementation.